### ISSUE TITLE
deprication of safe_load in ruamel-yaml

### ISSUE THREAD

**Issue by maxgeilert** [thegeeklab/ansible-doctor#661]
Hi,

wanted to use this tool, but it's not compatible with my dev environment. 
The ruamel yaml module deprecated some functions in v0.18.0, see https://pypi.org/project/ruamel.yaml/0.18.0.

**Comment by xoxys** [comment]
Can you please share some more information? What's the error you see? How have you installed ansible-doctor?

**Comment by maxgeilert** [comment]
Sorry, I thought I fixed it in my local copy and wanted to submit a PR, but I just found out that its already patched in git.

I installed ansible-doctor via pip inside a dev container. It installed version 3.1.1 successfully. In the local copy of config.py in line 271 it still has `file_dict = ruamel.yaml.safe_load(s)` instead of the new syntax.

Just figured out that a simple `python3 -m pip install --upgrade ansible-doctor` fixes my problem, sorry to bother you!
